astropy/constants/__init__.py

```diff
@@ -23,12 +24,47 @@
 except ImportError:
```

Comment:
I wonder if I have broken this initial import. In an interactive session, I have to `import astropy.units` before using a constant.

Reply:
Is this still an issue?

Reply:
Oh, this was something you had fixed by introducing `config.py`. Not an issue anymore. In fact the `try...except` does not appear to be needed.